packages/salesforcedx-vscode-core/test/vscode-integration/commands/forceConfigSet.test.ts

```diff
@@ -6,34 +6,34 @@
  */
 
 import { expect } from 'chai';
+import * as sinon from 'sinon';
+import * as vscode from 'vscode';
 import { ForceConfigSetExecutor } from '../../../src/commands';
-import { nls } from '../../../src/messages';
+
+const sandbox = sinon.createSandbox();
+let openTextDocumentSpy: sinon.SinonSpy;
 
 describe('Force Config Set', () => {
+  beforeEach(() => {
+    openTextDocumentSpy = sandbox.spy(vscode.workspace, 'openTextDocument');
+  });
+
+  afterEach(() => {
+    sandbox.restore();
+  });
+
   it('should build the force config set command', async () => {
     const usernameOrAlias = 'test-username1@gmail.com';
     const forceConfigSet = new ForceConfigSetExecutor(usernameOrAlias);
-    const forceConfigSetCommand = forceConfigSet.build({});
-    expect(forceConfigSetCommand.toCommand()).to.equal(
-      `sfdx force:config:set defaultusername=${usernameOrAlias}`
-    );
-    expect(forceConfigSetCommand.description).to.equal(
-      nls.localize('force_config_set_org_text')
-    );
+    sandbox.stub(forceConfigSet, 'run').returns(true);
+    expect(forceConfigSet.getUsernameOrAlias()).to.equal(usernameOrAlias);
   });
-});
 
-describe('Force Config Set using multiple aliases for a single username', () => {
```

Comment:
More notes while we're working through this - have you tested what happens when a user has multiple aliases for a single username?

Reply:
Yes, I've had multiple aliases in the past, and the expected behavior is that the first one is the one that gets set in the config. I still plan to keep this test, just modify it so we're making sure config is set with the correct alias.